dashboard/app/views/api/hoc_courses_teacher_guides.html.haml

```diff
@@ -1,18 +1,23 @@
 .row{style: "margin: 0;"}
   = render partial: 'shared/course_tall_block', locals: { id: Script::STARWARS_NAME, show_teacher_guide: true }
   = render partial: 'shared/course_tall_block', locals: { id: Script::FROZEN_NAME, show_teacher_guide: true }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::SPORTS_NAME }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::FLAPPY_NAME, show_teacher_guide: true }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::POEM_ART_2021_NAME }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::HELLO_WORLD_FOOD_2021_NAME }
```

Comment:
I just noticed that we also have `POEM_ART` and `HELLO_WORLD_FOOD` in `script_constants.rb`.  I suspect we should actually use these non-year strings.

Reply:
@breville ah good catch! Just updated everything.